styles/badge.css

```diff
@@ -0,0 +1,104 @@
+.badge_container-main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 1rem;
+}
+
+.text_with_badge {
+  display: flex; /*for centering text*/
+  align-items: center;
+  margin: 1rem;
+}
+
+.badge {
+  height: 1.5rem;
+  width: 1.5rem;
+  background-color: var(--status-busy);
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
used display:flex many times try not to repeat code make a dedicated class for flex and use it where ever u need

Reply:
Will work on this issue!

Reply:
DOne!!